### PR TITLE
Updated sankey visualization method to support generating sankeys for different regions

### PR DESCRIPTION
**Pull request by MAZamarripa**
… multiple regions

## Fixes/Addresses/Summary/Motivation:

(Replace this with the issue # fixed or addressed in this PR as well as any other related PRs.  If
no issue or PRs exist, then add a brief statement on what changes this PR is bringing in.)

## Changes proposed in this PR:
-
-

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

File: pareto/utilities/results.py [hunk 1]
```diff
@@ -967,7 +1033,7 @@ def plot_bars(input_data, args):
         )
 
         # Update animation settings
-        fig.layout.updatemenus[0].buttons[0].args[1]["frame"]["duration"] = 200
+        fig.layout.updatemenus[0].buttons[0].args[1]["frame"]["duration"] = 800
```
Comment (melody-shellman) [inline pareto/utilities/results.py:1036]: When I run the tests locally, they run and complete within 23min. When the tests run on github they do not finish in 35 minutes. I'm not really sure why ...does changing these figure layout affect build time ?

**Review by melody-shellman — COMMENTED** [review]
The tests are passing when running locally, but not on git. ... at least not within 35 min. Tests have typically run in <20min. Here are a few things that I can think of to reduce this time ....
-  I noticed that test_plot_scatter.py has 40 tests. Why do we need so many? Are all of these necessary?
- Do the changes to allow for input multi regions affect more than a few of these tests? Might this be why the testing time has increased drastically?
- Perhaps it's the "frame duration" setting that affects build time?

**Comment by melody-shellman** [comment]
One more thing - it may also be good to add the output files of the pytest in the git ignore. I think it is just these files:
        Region 1_first_sankey.html
        Region 2_first_sankey.html
        first_bar.html
        first_sankey.html
        first_scatter.html

File: pareto/utilities/results.py (hunk 1 above)
Comment (MAZamarripa) [reply pareto/utilities/results.py:1036]: @lbianchi-lbl @dangunter can you check this issue? it could be the iplot() method.

**Comment by melody-shellman** [comment]
This work will be completed in PR #149

**Pull request closed without merging**